Anthony/library_challenge.py

```diff
@@ -0,0 +1,219 @@
+    },
+    {
+        'title': 'BORN A CRIME',
+        'author': 'Trevor Noah',
+        'description': 'A memoir about growing up biracial in apartheid South Africa by the host of “The Daily Show.”'
+    },
+    {
+        'title': 'THE WARMTH OF OTHER SUNS',
+        'author': 'Isabel Wilkerson',
+        'description': 'An account of the Great Migration of 1915-70, in which six million African-Americans abandoned the South.'
+    },
+    {
+        'title': 'THE COLOR OF LAW',
+        'author': 'Richard Rothstein',
+        'description': 'A case for how the American government abetted racial segregation in metropolitan areas across the country.'
+    },
+    {
+        'title': 'THE NEW JIM CROW',
+        'author': 'Michelle Alexander',
+        'description': 'A law professor on the “war on drugs” and its role in the disproportionate incarceration of Black men.'
+    },
+    {
+        'title': 'THE TRUTHS WE HOLD',
+        'author': 'Kamala Harris',
+        'description': 'A memoir by the daughter of immigrants who is now a California senator and the 2020 Democratic candidate for vice president.'
+    },
+    {
+        'title': 'SAPIENS',
+        'author': 'Yuval Noah Harari',
+        'description': 'How Homo sapiens became Earth’s dominant species.'
+    },
+    {
+        'title': 'BRAIDING SWEETGRASS',
+        'author': 'Robin Wall Kimmerer',
+        'description': 'A botanist and member of the Citizen Potawatomi Nation espouses having an understanding and appreciation of plants and animals.'
+    },
+    {
+        'title': 'MY GRANDMOTHER\'S HANDS',
+        'author': 'Resmaa Menakem',
+        'description': 'A therapist who specializes in trauma, body-centered psychotherapy and violence prevention explains racism\'s effect on the body.'
+    },
+    {
+        'title': 'ON TYRANNY',
+        'author': 'Timothy Snyder',
+        'description': 'Twenty lessons from the 20th century about the course of tyranny.'
+    },
+    {
+        'title': 'THE ROAD TO UNFREEDOM',
+        'author': 'Timothy Snyder',
+        'description': 'Snyder explores Russian attempts to influence Western democracies and the influence of philosopher Ivan Ilyin on Russian President Vladimir Putin and the Russian Federation in general.'
+    }
+]
+# 2.0
+# Describe the structure of the data in books_with_details. What types of data are nested within others? How do you know?
+print()
+
+print(type(books_with_details)) 
+
+print()
+
+print(type(books_with_details[0])) #This indexes what is in the list and explains the type. They typs is the same for all items in the list
+
+print()
+# 2.1
+# Create a function 'count_books' that returns the number of books in the books_with_details list
+# Parameters: Not needed for this function
+# Return: number of books (integer)
+
+def count_books():
+    print(len(books_with_details))
+    return #return is used to stop the function
```

Comment:
Used to stop the function, but most importantly  return retains the output of your function